charts/karmada/templates/karmada-search.yaml

```diff
@@ -87,6 +90,14 @@ spec:
           resources:
           {{- toYaml .Values.apiServer.resources | nindent 12 }}
       volumes:
+        - name: etcd-cert
+          secret:
+          {{- if eq .Values.etcd.mode "internal" }}
+            secretName: {{ $name }}-cert
+          {{- end }}
+          {{- if eq .Values.etcd.mode "external" }}
+            secretName: external-etcd-cert
```

Comment:
Constant names don't seem like a good idea.

Reply:
https://github.com/Michael754267513/karmada/blob/master/charts/karmada/templates/karmada-apiserver.yaml#L147

Reply:
karmada chart does not mount external ETCD-related certificates when etcd is in external mode

Reply:
Constants are really bad, but the external etcd is unique and is configured according to apiserver

Reply:
> https://github.com/Michael754267513/karmada/blob/master/charts/karmada/templates/karmada-apiserver.yaml#L147

https://github.com/karmada-io/karmada/blob/5f96d733ba4b3ed80d68c2e1ade8ecfe2f1c9e77/charts/karmada/templates/karmada-aggregated-apiserver.yaml#L109

Reply:
> > https://github.com/Michael754267513/karmada/blob/master/charts/karmada/templates/karmada-apiserver.yaml#L147
> 
> https://github.com/karmada-io/karmada/blob/5f96d733ba4b3ed80d68c2e1ade8ecfe2f1c9e77/charts/karmada/templates/karmada-aggregated-apiserver.yaml#L109

  name: external-etcd-cert

Reply:
In secret, it's a constant declaration

Reply:
Thanks for your confirm that i have created a issue for it.